app/src/main/java/org/oppia/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -71,6 +83,7 @@ class ProfileChooserFragmentPresenter @Inject constructor(
     binding.profileRecyclerView.isNestedScrollingEnabled = false
     binding.profileRecyclerView.apply {
       adapter = createRecyclerViewAdapter()
+      layoutManager = profileLayoutMode
```

Comment:
We are already setting GridLayoutManager in XML file for Recyclerview. So you can remove this line and run the code.

Reply:
Isn't it better to remove the layoutManager declaration from the XML ? As we have to manage the LayoutManager via code.

Reply:
@Sarthak2601 based on new implementation removing layout manager from XML is correct approach.

Reply:
Done.